README.md

```diff
@@ -1,4 +1,51 @@
-# OpenEuropa Drupal module template
+# Config Owner
+
+The Config Owner module provides the ability for modules to control the configuration they ship with. That means preventing sites that use it from
+exporting/importing changes to these configurations.
+
+## How to use
+
+The module works by exposing a YML based plugin type called Owned Config. Any module that wants to "own" some configuration, needs to create such a plugin. 
+
+In the plugin, the "owned" configuration is referenced under the relevant keys:
+
+* `install` (configuration the module ships with)
+* `optional` (optional configuration the module ships with)
+* `owned` (owned configuration that the module does not ship with)
+
+These keys map to the location inside the module's `config` directory.
+
+See the `config_owner_test` module as an example of using this plugin.
```

Comment:
We should explain here how the plugin works as we cannot assume the test module to be self-explanatory. Also, at the moment the plugin syntax is quite intuitive but that might change once we factor in third party settings.